helm/values.yaml

```diff
@@ -27,3 +26,30 @@ deployer:
   namespace: bc5936-dev
   serviceAccount:
     enabled: true
+
+cas-postgres:
+  patroni:
+    resources:
+      limits:
+        cpu: "15m"
+        memory: "150Mi"
```

Comment:
@wenzowski I attempted to change these values, any time I went over the build fails.